sql/upgrade/vectors--0.2.1--0.3.0.sql

```diff
@@ -32,6 +32,12 @@ CREATE FUNCTION vectors."_vectors_veci8_operator_neq"(
 ) RETURNS bool
 IMMUTABLE STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_veci8_operator_neq_wrapper';
 
+CREATE  FUNCTION vectors."_vectors_veci8_operator_mul"(
```

Comment:
I think there should not be `vectors.` since schema can be override by `.schema` now.